docs/fundamentals/toc.yml

```diff
@@ -1056,10 +1068,16 @@ items:
               href: code-analysis/style-rules/pattern-matching-preferences.md
             - name: IDE0019
               href: code-analysis/style-rules/ide0019.md
-            - name: IDE0020
-              href: code-analysis/style-rules/ide0020.md
+            - name: IDE0020 and IDE0038
```

Comment:
Do we want to have separate entries for these in case IDE0025 comes along?

Reply:
Sounds good, let me add separate entries for such cases.